cmd/panfs.go

```diff
@@ -1953,16 +1952,17 @@ func (fs *PANFSObjects) getTempDir(bucketDir string) string {
 // function returns an error when new bucket path is on the top of any existing bucket path as well.
 // Example. Existing bucket paths: /a/b/c /a/b/d /a/b1/b2/c3/d4
 // /a/b/c/d /a/b/d/e /a/b1/b2/c3/d4/e5/f	INVALID
-// /a/b/c1 /a/b/f 							OK
+// /a/b/c1 /a/b/f							OK
 // /a/b  /a/b1/c3							INVALID
```

Comment:
Why is the `/a/b1/c3` path invalid?
We should probably add an example for this change we're making here.

Reply:
Yes, `/a/b1/c3` should be valid, thanks!